### DISCUSSION TITLE
Help Getting This Working On My Machine

### DISCUSSION THREAD

**Discussion by jrpetersjr** [henrikfroehling/Trakt.NET#414] [Help] [answered]
I've been trying to set this up on my local machine and nothing I'm doing is working. I'm running on Visual Studio 2022 and I've tried rebuilding the solution and restoring the nuget packages, but it's still not working. 

Let me know if you need to know any more information from me.  @henrikfroehling

**Comment by henrikfroehling** [comment]
Hello,

in order to provide helpful steps I need more information.

Which branch did you checkout?

Do you get any errors and if so, which errors?

Which version of VS 2022 do you use? Stable or Preview?

What exactly is not working?
The build process, restoring the NuGet dependencies or something else?

And just to rule out if it's a general problem or specific to the library:
Is it possible to create a new and clean project and building it?

**Comment by jrpetersjr** [comment]
I cloned the Release-2.0.0 branch.  My verison of visual Studio 2022. 
![image](https://user-images.githubusercontent.com/9285616/225435571-9fbb846f-627f-425b-bdf2-8b6c4fc4da8c.png)

I'm getting a ton of errors. 159 errors to be exact. I get them when building.

![image](https://user-images.githubusercontent.com/9285616/225435910-e2bd014a-decd-48ac-ad2c-63af6157350f.png)
![image](https://user-images.githubusercontent.com/9285616/225437680-bc152eaf-dd01-4504-9875-07cce4318294.png)
![image](https://user-images.githubusercontent.com/9285616/225437835-bb8eeaa5-e615-483a-a497-f1322b2a0784.png)
![image](https://user-images.githubusercontent.com/9285616/225437976-f56c7c84-9b04-4417-914b-0f86e53ccb54.png)


Below is what the TraktClient looks like.
![image](https://user-images.githubusercontent.com/9285616/225436180-dd6f8c2c-8932-43a5-89c0-b0be54af5677.png)

Here's what is looks like when I try to restore the Nuget packages.
![image](https://user-images.githubusercontent.com/9285616/225438497-139422f5-0a2b-4e1d-9ed6-46bae519bba0.png)


I can make a new project and build it and it works. 
![image](https://user-images.githubusercontent.com/9285616/225439377-daa2be4d-159e-48de-961e-e74e873183ad.png)


Thanks For Your Help In Advance.

**Comment by jrpetersjr** [comment, marked as answer]
Ok. I figured it out. My visual studio didn't have Nuget.org set as a package manager and everything was looking locally. Sorry for wasting your time.

**Reply by henrikfroehling** [reply]
Okay, no problem. ☺️
